modules/cnv-pxe-booting-with-mac-address.adoc

```diff
@@ -0,0 +1,149 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv-configuring-pxe-booting.adoc
+
+[id="cnv-pxe-booting-with-mac-address-{context}"]
+= PXE booting with a specified MAC address
+
+PXE booting, or network booting, is supported in {ProductName}.
+Network booting allows a computer to boot and load an
+operating system or other program without requiring a locally attached
+storage device. For example, you can use it to choose your desired OS
+image from a PXE server when deploying a new host.
+
```

Comment:
I'm not sure, but it might be worth adding a comment, saying that this doc only instructs how to set up the PXE client side, not the server side, which is not (necessarily) related to CNV.